results/benchmarks.json

```diff
@@ -720,4 +720,4 @@
         "warmup_time": -1
     },
     "version": 2
-}
\ No newline at end of file
+}
```

Comment:
Ah. I see the spaces are being added by pre-commit vs the asv style. 

Can we run the linter  in the action before committing the results?